### ISSUE TITLE
[genie; lamp] - Install without toltec - reMarcable2

### ISSUE THREAD

**Issue by PepikVaio** [rmkit-dev/rmkit#229]
How to install "genie" and "lamp" without toltec please?

Remarkable 2.
Version 3.13.x

Thank you.

**Comment by PepikVaio** [comment]
I managed to install genie on rm2 without toltec.
However, sometimes I get this error.

```
Aug 05 21:08:41 reMarkable genie[410]: RUNNING COMMAND /opt/bin/lamp < /home/root/swipe_left.in
Aug 05 21:08:41 reMarkable genie[410]: malloc(): mismatching next->prev_size (unsorted)
Aug 05 21:08:41 reMarkable genie[410]: CLEANING UP FB
Aug 05 21:08:41 reMarkable genie[410]: received SIGABRT, exiting
```

**Comment by Eeems** [comment]
> I managed to install genie on rm2 without toltec. However, sometimes I get this error.
> 
> ```
> Aug 05 21:08:41 reMarkable genie[410]: RUNNING COMMAND /opt/bin/lamp < /home/root/swipe_left.in
> Aug 05 21:08:41 reMarkable genie[410]: malloc(): mismatching next->prev_size (unsorted)
> Aug 05 21:08:41 reMarkable genie[410]: CLEANING UP FB
> Aug 05 21:08:41 reMarkable genie[410]: received SIGABRT, exiting
> ```

This looks to be a different issue than this was first opened for. A new issue with reproduction steps will be required.

**Comment by PepikVaio** [comment]
I didn't want to create another issue unnecessarily (the first issue is almost solved).
Is it enough to rewrite only the name?

**Comment by raisjn** [comment]
yes, it's fine to rewrite the name.  can you share what is in the input file?
